.github/workflows/py-tests.yml

```diff
@@ -45,6 +45,8 @@ jobs:
         run: pip3 install LinguaFrancaBase
       - name: Install Google API Python Client
         run: pip3 install --upgrade google-api-python-client
+      - name: Install custom serializer test
+        run: pip3 install -e ./test/Python/src/serialization/pickle_serializer
```

Comment:
Yes I can move the install command into the preamble. I'll use `lf.source_directory()` to find the package location. Though it shouldn't be too hard to run test locally, just run the pip install command before running the tests.

Reply:
Because of #2378, installing the custom serializer in the preamble is not feasible with a relative path. If we install the package with absolute path, it would be even harder to run the test locally, because the absolute path is sure to be different in the local environment. 

I think the solution here is to install the custom package first. To run the test locally, run `pip3 install -e ./test/Python/src/serialization/pickle_serializer` in the project root directory. I'll add a comment in the test file.

Reply:
Why does the package need to be installed? Can we not simply put the external Python file into the `files` property and then use a relative import to import the module?

Reply:
I moved the install to the preamble so that the tests are self-contained.